packages/wrangler/src/utils/print-bindings.ts

```diff
@@ -288,6 +290,18 @@ export function printBindings(
 		});
 	}
 
+	if (secret_stores !== undefined && secret_stores.length > 0) {
+		output.push({
+			name: friendlyBindingNames.secret_stores,
+			entries: secret_stores.map(({ binding, store_id, name }) => {
+				return {
+					key: binding,
+					value: addSuffix(`${store_id}/${name}`),
```

Comment:
This decides how the binding is printed in the terminal when you run wrangler dev / deploy. This is how it looks like right now:

```
Your worker has access to the following bindings:
- Secret Stores:
	- SECRET: store_id/secret_name
```